classes/main.php

```diff
@@ -48,7 +55,7 @@ protected function initialize() {
 		}
 
 		// Make sure the options property is always current.
-		add_action( 'init', [ 'WPSEO_Option_Woo', 'register_option' ] );
+		add_action( 'init', [ 'Yoast\WP\Woocommerce\Classes\Options', 'register_option' ] );
```

Comment:
```suggestion
		add_action( 'init', [ Options::class', 'register_option' ] );
```

:heavy_plus_sign:  near the top of the file:
```php
use Yoast\WP\Woocommerce\Classes\Options;
```

This is not _strictly_ necessary as it's in the same namespace, but it's good practice to be explicit about all classes used in the file.